### ISSUE TITLE
Log spam about TrajectoriesDebug

### ISSUE THREAD

**Issue by puetzk** [neuoy/KSPTrajectories#79]
```
[LOG 08:47:04.493] PartLoader: Compiling Part '...'
[ERR 08:47:04.508] Cannot find a PartModule of typename 'TrajectoriesDebug'
```
repeated for every single part.

This TrajectoriesDebug PartModule is only included `#if DEBUG`, so it would be nice if  the ModuleManager setup in releases could either comment this out or somehow detect that it's not present (maybe just do :NEEDS[TrajectoriesDebug] and make a dummy folder or something)?

**Comment by neuoy** [comment]
Indeed, I've not been very smart at removing debugging stuff from the release builds. I guess the best way, for loading performances, is to remove it completely from the config file ; I'll do that in the next release.

**Comment by neuoy** [comment]
fixed in 1.6.5
